drizzlepac/alignimages.py

```diff
@@ -486,11 +504,52 @@ def run_align(input_list, archive=False, clobber=False, debug=False, update_hdr_
         result.add_column(filteredTable[col], name=col)
     filteredTable.pprint(max_width=-1)
 
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+def match_relative_fit(imglist, reference_catalog,group_id_dict):
+    """Perform cross-matching and final fit using 2dHistogram matching
+
+    Parameters
+    ----------
+    imglist : list
+        List of input image `~tweakwcs.tpwcs.FITSWCS` objects with metadata and source catalogs
+
+    reference_catalog : Table
+        Astropy Table of reference sources for this field
+
+    Returns
+    --------
+    imglist : list
+        List of input image `~tweakwcs.tpwcs.FITSWCS` objects with metadata and source catalogs
+
+    """
+    # Specify matching algorithm to use
+    #restore group IDs to their pristine state prior to each run.
+    for image in imglist:
+        image.meta["group_id"] = group_id_dict["{}_{}".format(image.meta["filename"],image.meta["chip"])]
+    match = tweakwcs.TPMatch(searchrad=75, separation=0.1,
+                             tolerance=2, use2dhist=True)
+    # match = tweakwcs.TPMatch(searchrad=250, separation=0.1,
+    #                          tolerance=100, use2dhist=False)
+
+    # Align images and correct WCS
+    #  Perform relative alignment
+    tweakwcs.align_wcs(imglist, None, match=match, expand_refcat=True)
+    for image in imglist:
+        image.meta["group_id"] = 1234567
+    #  Perform absolute alignment
```

Comment:
Add a comments here around the calls to tweakwcs.align_wcs.  In my version I had added before the first call
    
    # Align images and correct WCS - this invocation does not use an astrometric catalog. 
    # This call allows all the input images to be aligned in a relative way using the first
    # input image as the reference.

Then before re-setting the group_id:

    # Set all the group_id values to be the same so the various images/chips will
    # be aligned to the astrometric reference catalog as an ensemble
    # BEWARE: If additional iterations of solutions are to be done, the group_id values need to be restored.

Reply:
Issue resolved in commit [d92cfae](https://github.com/spacetelescope/drizzlepac/pull/268/commits/d92cfaeb1147679837175dc7610c4a355c406d3a)